modules/cram/straglr.nf

```diff
@@ -0,0 +1,32 @@
+process straglr_call {
```

Comment:
consistency: add `label 'straglr_call'` so resources can be assigned by label just like other processes.

Reply:
fixed (different line)